x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/waterfall_helpers/__test__/waterfall_helpers.test.ts

```diff
@@ -148,6 +148,98 @@ describe('waterfall_helpers', () => {
     });
   });
 
+  describe('getWaterfall - legacy', () => {
```

Comment:
Instead of "legacy" can we be more specific about what use case this is testing? Eg "When trace items don't have `parent.id` (pre 6.5 behaviour)"